py3status/modules/imap.py

```diff
@@ -14,6 +14,8 @@
     security: login authentication method: 'ssl' or 'starttls'
         (startssl needs python 3.2 or later) (default 'ssl')
     server: server to connect (default None)
+    use_idle: use IMAP4 IDLE instead of polling; requires imaplib2,
+        compatible server; disables cache_timeout (default: False)
```

Comment:
I think we should just do this if we can rather than bothering the user

@ultrabug what do you think?

Reply:
not too easy to do. only way would be to wait for `IMAP.error` to get thrown, then parse the error string and switch `use_idle` off. I'd just keep it as-is.